src/Apps/Artist/Routes/Overview/Components/ArtworkFilter/index.tsx

```diff
@@ -311,13 +308,14 @@ class Filter extends Component<Props> {
                       </Media>
 
                       {/* Main Artwork Grid */}
-                      <Box width={xs ? "100%" : "75%"}>
-                        {!hideTopBorder && <Separator mb={2} mt={-1} />}
+                      <Box width={["100%", "75%"]}>
+                        <Media greaterThan="xs">
+                          {!hideTopBorder && <Separator mb={2} mt={-1} />}
+                        </Media>
```

Comment:
I did a few more changes in this section, so you’ll probably run into a merge conflict… sorry about that!